repository/redis/operator/templates/init.yaml

```diff
@@ -2,11 +2,15 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   namespace: {{ .Namespace }}
-  name: {{ .PlanName }}-job
+  name: {{ .PlanName }}-{{ .Name }}-{{ .OperatorName }}-job
+  labels:
+    app: redis
+    redis: {{ .OperatorName }}
+    instance: {{ .Name }}
```

Comment:
should we also add `plan: {{ .PlanName }}` as label?

Reply:
I think so. Done!